crates/ide_assists/src/handlers/auto_import.rs

```diff
@@ -99,8 +99,12 @@ pub(crate) fn auto_import(acc: &mut Assists, ctx: &AssistContext) -> Option<()>
             format!("Import `{}`", &import),
             range,
             |builder| {
-                let rewriter =
-                    insert_use(&scope, mod_path_to_ast(&import), ctx.config.insert_use.merge);
+                let rewriter = insert_use(
+                    &scope,
+                    mod_path_to_ast(&import),
+                    ctx.config.insert_use.merge,
```

Comment:
It might make sense to pass `&ctx.config.insert_use` instead, see https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#functions-with-many-parameters.